app/scripts/app.js

```diff
@@ -32,10 +32,19 @@ blocList.controller('Landing.controller', ['$scope', '$firebaseArray', function(
         }
         $scope.todos.$add({
             title: newTodo,
-            completed: false
+            completed: false,
+            submitted: Date.now(),
+            expiryDate: Date.now() + 604800000
```

Comment:
This is a [magic number](http://en.wikipedia.org/wiki/Magic_number_%28programming%29). Maybe consider naming it?